todo/src/server.ts

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+import { Server, ResponseToolkit } from "@hapi/hapi";
+import { getRoute } from "./routes/routes";
+
+
+export let server: Server;
+
+export const init = async () => {
+    server = new Server({
+        port: 5000,
+        host: 'localhost'
+    });
+
+    getRoute();
```

Comment:
See here you could just export the array of routes from your routes file and then pass it into `server.route` here